sig-apps/annual-report-2021.md

```diff
@@ -0,0 +1,60 @@
+# Operational
+How are you doing with operational tasks in sig-governance.md?
+Is your README accurate? have a CONTRIBUTING.md file?  
+Yes, both the [README.md](https://github.com/kubernetes/community/blob/master/sig-apps/README.md)
+and the [CONTRIBUTING.md](https://github.com/kubernetes/community/blob/master/sig-apps/CONTRIBUTING.md)
+are up-to-date.
+
+All subprojects correctly mapped and listed in sigs.yaml?  
+Yes
+
+What’s your meeting culture? Large/small, active/quiet, learnings? Meeting notes up to date? Are you keeping recordings up to date/trends in community members watching recordings?  
+We hold small and active biweekly meeting. The call is divided into few sections, such as important announcements, then demos and finally discussions on the current topics. If time allows, we try to review issues and pull requests. All of the meetings are recorded and available online, all the meeting invites are up-to-date and present in community calendar.
+
+
+How does the group get updates, reports, or feedback from subprojects? Are there any springing up or being retired? Are OWNERS files up to date in these areas?  
+Every subproject is free to provide an update at our biweekly meeting, but we don’t enforce that specifically. OWNERS files are up-to-date.
```

Comment:
Do you happen to do any check on subproject activity? Are there any that you think should be archived or flagged as maintenance mode?